tanstack-router-file-based/src/routeTree.gen.ts

```diff
@@ -0,0 +1,243 @@
+/* eslint-disable */
```

Comment:
is this file supposed to be .gitignored. Im not sure off hand

Reply:
it's okay to have it

Reply:
like the lockfile it's generated, but doesn't matter